bundle/Resources/views/themes/ngadmin/content/location_view.html.twig

```diff
@@ -26,6 +26,12 @@
                     </svg>
                 </a>
                 <a href="{{ url }}">{{ content_type.name }}</a>
+
+                {% set array = [] %}
+                {% for language in content.versionInfo.languages %}
+                    {% set array = array|merge([language.languageCode|show_language_name]) %}
+                {% endfor %}
+                &nbsp {{array|join(', ') }}
```

Comment:
`&nbsp` is not needed now.

Reply:
It is IMO.

Without:
![image](https://github.com/netgen/better-ibexa-admin-ui/assets/99658228/b1c0eb5f-1c49-4a34-af5b-c9215e43dc3f)

With:
![image](https://github.com/netgen/better-ibexa-admin-ui/assets/99658228/9be81a5a-169b-4382-8713-34249ecfb043)

Reply:
Why not add the list of languages below the content type name?

Besides, `&nbsp` should really be `&nbsp;` (with a semicolon).

Reply:
Idk, flags were in the same row. It can be either way I guess: This is how it was done originally so I didn't change it.